packages/storybook/stories/INTERNAL/TroubleShootingGuideErrorBar/TroubleShootingGuideErrorBar.stories.tsx

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { mergeStyles } from '@fluentui/react';
+import { useTheme, _TroubleShootingGuideErrorBar } from '@internal/react-components';
+import { Meta } from '@storybook/react/types-6-0';
+import React from 'react';
+import {
+  _TroubleShootingGuideErrorBarProps,
+  _TroubleShootingGuideErrorBarStrings
+} from '../../../../react-components/src/components/TroubleShootingGuideErrorBar';
+import { COMPONENT_FOLDER_PREFIX } from '../../constants';
+import { controlsToAdd, hiddenControl } from '../../controlsUtils';
+
+const TroubleShootingGuideErrorBarStory = (args): JSX.Element => {
+  const theme = useTheme();
+  const onPermissionsTroubleshootingClick = (permissionState: {
+    camera: PermissionState;
+    microphone: PermissionState;
+  }): void => {
+    console.log(permissionState);
+  };
+
+  const onNetworkingTroubleShootingClick = (): void => {
```

Comment:
troubleshooting is one word, so onNetworkingTroubleShootingClick -> onNetworkingTroubleshootingClick